docs/docsite/rst/user_guide/vault.rst

```diff
@@ -241,7 +241,7 @@ To encrypt a string read from stdin and name it 'db_password':
 
 Result::
 
-    Reading plaintext input from stdin. (ctrl-d to end input)
+    Reading plaintext input from stdin. (ctrl-d twice to end input)
```

Comment:
ammend to `(ctrl-d to end input, x2 if your content does not already have a new line)`

Reply:
addressed in 1e06c000b49700a52505cfff6f74e8f4a9864b62